MSAL/src/MSALResult.m

```diff
@@ -113,7 +133,13 @@ + (MSALResult *)resultWithMSIDTokenResult:(MSIDTokenResult *)tokenResult
         account.accountClaims = claims.jsonDictionary;
     }
     
-    return [self resultWithAccessToken:tokenResult.accessToken.accessToken
+    NSString *accessToken = [authScheme getSecret:tokenResult.accessToken popManager:popManager error:error];
+    if (!accessToken)
+    {
+        return nil;
```

Comment:
nit: log

Reply:
Logging will be done in the scheme class for this case.